### PR TITLE
Pull request for Rvfbrnet 36

### PR DESCRIPTION
**Pull request by AndriiKatsubo**

### CODE REVIEW

**Comment by KostetskyiYaroslav** [comment]
# **Do you want to push all this commits to develop branch?**
'Merged branch develop into current branch.' - that commit shoudn't be in your pull request, as i know. Am i right?
https://i.stack.imgur.com/n3gDy.png
One branche = One feature

**Comment by AndriiKatsubo** [comment]
I have merged develop branch into this branch because of next reasons:
1) This feature uses features from develop branch;
2) After such merge next merge of this branch with develop would go on more easily.
As I understand, that scheme doesn't prohibit me from doing such merges.

File: MedicalDocManagment.WebUI/app/http.service.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/toPromise';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/observable/throw';
+
+import UserModel from './models/usermodel';
+
+@Injectable()
+export class HttpService {
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/http.service.ts]: Rename class name

File: MedicalDocManagment.WebUI/app/http.service.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  private http_: Http;
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/http.service.ts]: move `_` napered

File: MedicalDocManagment.WebUI/app/http.service.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+  constructor(http: Http) {
+    this.http_ = http;
+  }
+
+  postData(obj: UserModel) {
+    const body = JSON.stringify(obj);
+    let headers = new Headers({ 'Content-Type': 'application/json;charset=utf-8' });
+    return this.http_.post('/api/Admin/AddUser', body, { headers: headers })
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/http.service.ts]: format methods list

File: MedicalDocManagment.WebUI/app/app.component.ts [hunk 4]
```diff
@@ -1,11 +1,21 @@
...
+
+import { UsersListComponent } from './userslist.component';
 
 @Component({
-        selector: 'my-app',
-        template: `<h1>Добро пожаловать {{name}}!</h1>
-                <label>Введите имя:</label>
-                <input [(ngModel)]="name" placeholder="name">`
+  selector: 'my-app',
+  template: '<h1>Добро пожаловать {{name}}!</h1>'+
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/app.component.ts]: separate template into another file `app.component.html`

File: MedicalDocManagment.WebUI/app/app.component.ts [hunk 5]
```diff
@@ -1,11 +1,21 @@
...
+    '<hr>'+
+    '<usersList>Завантажується список користувачів...</usersList>'+
+    '<hr>'+
+    '<app-user-add></app-user-add>'
 })
 export class AppComponent {
-        name = '';
-}
\ No newline at end of file
+  name = '';
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/app.component.ts]: use types anonation

File: MedicalDocManagment.WebUI/app/models/positionmodel.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * Class, which represents data model for position of the user.
+ */
+export default class PositionModel {
+  /**
+   * Unique identifier of position.
+   * @type {number}
+   */
+  id: number;
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/models/positionmodel.ts]: change type on `string`

File: MedicalDocManagment.WebUI/app/userslist-item-actionlist.component.ts [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+ * @fileoverview This file defines UsersListItemActionListComponent — component, which implements
+ *     list of actions on item from list of users.
+ * @author andriy_katsubo@ukr.net (Andriy Katsubo)
+ */
+import { Component, Input } from '@angular/core';
+import { Response } from '@angular/http';
+
+import UserModel from './models/usermodel'
+
+@Component({
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/userslist-item-actionlist.component.ts]: add `moduleId` prop

**Review by KostetskyiYaroslav — APPROVED** [review]
Approve with minor comments

**Pull request merged**